tools/flaky-test-reporter/config.go

```diff
@@ -39,10 +39,12 @@ var (
 	jobConfigs = map[string][]JobConfig{
 		// CI flows for serving repo
 		"serving": {{Name: "ci-knative-serving-continuous", Type: prow.PostsubmitJob, SkipGithubIssue: false},
-			{Name: "ci-knative-serving-istio-1.0.7-mesh", Type: prow.PostsubmitJob, SkipGithubIssue: true},
-			{Name: "ci-knative-serving-istio-1.0.7-no-mesh", Type: prow.PostsubmitJob, SkipGithubIssue: true},
-			{Name: "ci-knative-serving-istio-1.1.7-mesh", Type: prow.PostsubmitJob, SkipGithubIssue: true},
-			{Name: "ci-knative-serving-istio-1.1.7-no-mesh", Type: prow.PostsubmitJob, SkipGithubIssue: true},
+			{Name: "ci-knative-serving-istio-1.0-mesh", Type: prow.PostsubmitJob, SkipGithubIssue: true},
```

Comment:
Wow. This definitely should be a dynamic config parser... Someday.

Reply:
will be